examples/example_keys.go

```diff
@@ -0,0 +1,39 @@
+/*
+Package examples - key constants to be used in the examples and utility function for generating keys
+*/
+package examples
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bitcoin-sv/spv-wallet-go-client/xpriv"
+)
+
+const (
+	// ExampleAdminKey - example admin key
+	ExampleAdminKey string = "xprv9s21ZrQH143K3CbJXirfrtpLvhT3Vgusdo8coBritQ3rcS7Jy7sxWhatuxG5h2y1Cqj8FKmPp69536gmjYRpfga2MJdsGyBsnB12E19CESK"
+
+	// you can generate new keys using `task generate-keys`
+
+	// ExampleXPriv - example private key
+	ExampleXPriv string = ""
+	// ExampleXPub - example public key
+	ExampleXPub string = ""
+
+	// ExamplePaymail - example Paymail address
+	ExamplePaymail string = ""
+)
+
+/*
+GenerateKeys - function for generating keys (private and public)
+*/
```

Comment:
In all our golang projects we have the convention - than functions/methods description are oneliners. Could you adjust it in your code?
```suggestion
//GenerateKeys - function for generating keys (private and public)
```

Reply:
done